### PR TITLE
enabling ibm_powervc events

### PR DESCRIPTION
**Pull request by Guddetisandeep**
Enabling IBM PowerVC events

### CODE REVIEW

File: managing_providers/cloud_providers/_topics/enabling_ibm_power_vc_events.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+## Enabling Events in IBM PowerVC
+
+
+To get the IBM PowerVC Events:
+
+1.  Login to the PowerVC CLI and check if the following services are enabled:
+    ```
+    powervc-config image audit
+    powervc-config image audit
```
Comment (jaywcarman) [inline managing_providers/cloud_providers/_topics/enabling_ibm_power_vc_events.md:9]: typo?
```suggestion
    powervc-config compute audit
```

File: managing_providers/cloud_providers/_topics/enabling_ibm_power_vc_events.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    ```
+
+2.  If the services are not enabled, use the following commands to enable them:
+    ```
+    powervc-config image audit --enable --restart
+    powervc-config compute audit --enable --restart
+    powervc-config storage audit --enable --restart
+    ```
+
+3.  After enabling the services, you can perform operations for VMs, storage, images, etc.
```
[the remark this answers is not in the document]
Comment (Guddetisandeep) [reply managing_providers/cloud_providers/_topics/enabling_ibm_power_vc_events.md:20]: Yeah, I hope first two are enough to get the events from PowerVC and view it on ManageIQ UI or API. I shall remove the remaining points.

File: managing_providers/cloud_providers/_topics/enabling_ibm_power_vc_events.md (hunk 1 above)
Comment (Guddetisandeep) [reply managing_providers/cloud_providers/_topics/enabling_ibm_power_vc_events.md:9]: Yes, I will Update the PR

File: managing_providers/cloud_providers/_topics/enabling_ibm_power_vc_events.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
+## Enabling Events in IBM PowerVC
+
+
+To get the IBM PowerVC Events:
```
Comment (jaywcarman) [inline managing_providers/cloud_providers/_topics/enabling_ibm_power_vc_events.md:4]:
```suggestion
Service events must be enabled in IBM PowerVC in order for {{ site.data.product.title_short }} to collect and display them.
```

File: managing_providers/cloud_providers/_topics/enabling_ibm_power_vc_events.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+    ```
+
+2.  If the services are not enabled, use the following commands to enable them:
+    ```
+    powervc-config image audit --enable --restart
+    powervc-config compute audit --enable --restart
+    powervc-config storage audit --enable --restart
+    ```
+
+3.  After enabling the services, you can perform operations for VMs, storage, images, etc.
```
Comment (jaywcarman) [inline managing_providers/cloud_providers/_topics/enabling_ibm_power_vc_events.md:20]: I'm a little confused by the "you can perform operations for VMs, storage, images, etc." part of this. If it's not part of the process to enable the services let's remove it or drop it from the enumeration:
```suggestion
After enabling the services {{ site.data.product.title_short }} will be able to collect and display event data.
```

**Review by jaywcarman — COMMENTED** [review]
Looking good. I have just a few more minor suggestions.

**Review by jaywcarman — APPROVED** [review]
LGTM. Thanks @Guddetisandeep !

**Review by agrare — APPROVED** [review]

**Pull request merged**